packages/hurl/src/http/client.rs

```diff
@@ -721,6 +724,16 @@ impl Client {
         headers
     }
 
+    /// Parse primary IP into `http::response_ip::IpAddr`
+    fn parse_primary_ip(&mut self) -> Result<IpAddr, HttpError> {
```

Comment:
I would change `parse_primary_ip` name to just `primary_ip` as there is  no parsing involved here, it's just a call to libcurl.